src/i18n/strings/en_EN.json

```diff
@@ -912,7 +913,6 @@
     "You've previously used a newer version of Riot on %(host)s. To use this version again with end to end encryption, you will need to sign out and back in again. ": "You've previously used a newer version of Riot on %(host)s. To use this version again with end to end encryption, you will need to sign out and back in again. ",
     "Incompatible Database": "Incompatible Database",
     "Continue With Encryption Disabled": "Continue With Encryption Disabled",
-    "Failed to indicate account erasure": "Failed to indicate account erasure",
```

Comment:
This was something that was missed in a previous PR. Might as well include it here so it is out of the way.